create_pr_quick_start_guide.rst

```diff
@@ -226,7 +226,14 @@ Click ``Create pull request``.
 
 21. Add a `changelog fragment <https://docs.ansible.com/ansible/devel/community/development_process.html#changelogs>`_ to the ``changelog/fragments`` directory. It will be published in release notes, so users will know about the fix.
 
-Commit and push it:
+Run sanity test for the fragment:
+
+.. code:: bash
+
+  ansible-test sanity changelog/fragments/myfragment.yml --docker -v
```

Comment:
One possible improvement might be:
```suggestion
  ansible-test sanity changelog/fragments/ --docker -v
```
that way you don't have to write the fragment's name again.

Reply:
@felixfontein thanks for the feedback! Sounds good to me.
@kbudde would you like to apply the suggestion  or we can merge the PR?